custom/icds_reports/utils/fix_child_health_missing_month.sql

```diff
@@ -0,0 +1,249 @@
+UPDATE child_health_monthly child_health
+ SET
+ 	current_month_stunting = CASE
```

Comment:
hey it looks these the stunting and wasting on the mnc page does not take value from these columns. can you verify and please update the correct columns.
it looks stunting and wasting on MNC page is taken from
```
zscore_grading_hfa
zscore_grading_hfa_recorded_in_month
zscore_grading_wfh
zscore_grading_wfh_recorded_in_month

```

Reply:
also you will also need to update the agg_child_health query accordingly